src/main/java/org/broadinstitute/hellbender/tools/dataflow/transforms/bqsr/BaseRecalibratorFn.java

```diff
@@ -191,10 +200,16 @@ private void apply(GATKRead originalRead, final ReferenceDataSource refDS, Itera
         }
 
         // We've checked in onTraversalStart() that we have a reference, so ref.get() is safe
-        final int[] isSNP = calculateIsSNP(read, refDS);
-        final int[] isInsertion = calculateIsIndel(read, EventType.BASE_INSERTION);
-        final int[] isDeletion = calculateIsIndel(read, EventType.BASE_DELETION);
-        final int nErrors = nEvents(isSNP, isInsertion, isDeletion);
+        //final int[] isSNP = calculateIsSNP(read, refDS);
+        //final int[] isInsertion = calculateIsIndel(read, EventType.BASE_INSERTION);
+        //final int[] isDeletion = calculateIsIndel(read, EventType.BASE_DELETION);
+        //final int nErrors = nEvents(isSNP, isInsertion, isDeletion);
```

Comment:
grouping those functions together should give us a tiny little bit of performance. I would like this preserved, though I haven't measured the performance impact yet.